src/common/Modal/LinkCopyModalForm.tsx

```diff
@@ -0,0 +1,31 @@
+import styled from 'styled-components';
+import { IcLinkWhite } from '../../assets';
+
+const LinkCopyModal = () => {
+  return (
+    <LinkCopyModalContainer>
+      <IcLinkWhite />
+      <Text>링크가 복사되었습니다</Text>
+    </LinkCopyModalContainer>
+  );
+};
+
+export default LinkCopyModal;
+
+const LinkCopyModalContainer = styled.article`
```

Comment:
ASK : 저같으면 div 로 했을거같은데 잘봤습니다 ! 혹시 article 로 해주신 이유가 있나요 ? 
```suggestion
const LinkCopyModalContainer = styled.div`
```

Reply:
article은 보통 독립적으로 구분할 수 있는 내용을 나타낸다고 해요!
요 컴포넌트 내에서 해당 부분이 독립적으로 구분가능한 부분이라고 생각해서 article로 구현했습니다 ~